components/docs-chef-io/static/automate-api-docs/all-apis.swagger.json

```diff
@@ -19,7 +19,7 @@
     },
     "license": {
       "name": "Apache 2.0",
-      "url": "https://github.com/chef/automate/blob/master/LICENSE"
+      "url": "https://github.com/chef/automate/blob/main/LICENSE"
```

Comment:
We can't change this by hand, but it should update the next time we regenerate the API files.